product-service/handlers/getAllProducts.ts

```diff
@@ -0,0 +1,29 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import 'source-map-support/register';
+import productList from '../productList.json';
+
+export const getAllProducts: APIGatewayProxyHandler = async (
+	event,
+	_context
+) => {
+	function getAsync() {
+		return new Promise((resolve) => setTimeout(() => resolve(productList), 0));
+	}
+
+	return {
+		statusCode: 200,
+		headers: {
```

Comment:
I'd suggest you to revisit your indentation settings
![image](https://user-images.githubusercontent.com/14991709/98071459-a51b4700-1e74-11eb-9e7f-e0906f8cefb1.png)

Reply:
and use prettier